jazzy_gz_macos.repos

```diff
@@ -0,0 +1,37 @@
+repositories:
+  ardupilot:
+    type: git
+    url: https://github.com/ArduPilot/ardupilot.git
+    version: master
+  ardupilot_gazebo:
+    type: git
+    url: https://github.com/ArduPilot/ardupilot_gazebo.git
+    version: ros2
+  ardupilot_gz:
+    type: git
+    url: https://github.com/ArduPilot/ardupilot_gz.git
+    version: main
+  ardupilot_sitl_models:
+    type: git
+    url: https://github.com/ArduPilot/SITL_Models.git
+    version: main
+  gps_umd:
+    type: git
+    url: https://github.com/swri-robotics/gps_umd.git
+    version: ros2-devel
+  micro_ros_agent:
+    type: git
+    url: https://github.com/micro-ROS/micro-ROS-Agent
+    version: jazzy
```

Comment:
The macOS humble version required a specific branch from your fork @srmainwaring. Can you check if that is still necessary on `Jazzy` ?

Reply:
I created `jazzy` and `humble` branches here. Now, you can edit the `ros2_gz.repos` directly